<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>666</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>710</width>
      <height>481</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="F_Floor">
     <attribute name="title">
      <string>1 Floor</string>
     </attribute>
     <widget class="QLabel" name="label_1">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>700</width>
        <height>400</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="S_Floor">
     <attribute name="title">
      <string>2 Floor</string>
     </attribute>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>671</width>
        <height>471</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>530</y>
      <width>701</width>
      <height>70</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>770</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCreate">
    <property name="title">
     <string>Create</string>
    </property>
    <widget class="QMenu" name="menuLamp">
     <property name="title">
      <string>Lamp</string>
     </property>
     <addaction name="actionSimple_Lamp"/>
     <addaction name="actionComplex_Lamp"/>
    </widget>
    <addaction name="menuLamp"/>
    <addaction name="actionJalousie"/>
    <addaction name="actionVentilator"/>
    <addaction name="actionDoor"/>
    <addaction name="actionBattery"/>
   </widget>
   <addaction name="menuCreate"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionJalousie">
   <property name="text">
    <string>Jalousie</string>
   </property>
  </action>
  <action name="actionSimple_Lamp">
   <property name="text">
    <string>Simple Lamp</string>
   </property>
  </action>
  <action name="actionComplex_Lamp">
   <property name="text">
    <string>Complex Lamp</string>
   </property>
  </action>
  <action name="actionVentilator">
   <property name="text">
    <string>Ventilator</string>
   </property>
  </action>
  <action name="actionDoor">
   <property name="text">
    <string>Door</string>
   </property>
  </action>
  <action name="actionBattery">
   <property name="text">
    <string>Battery</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
